.github/actions/npm-cache/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Setup npm cache'
```

Comment:
Most everything in this action is already covered by the `setup-node` action caching all the packages in the OS level cache and a call to `npm run compile`.